src/lib/holocene/button.svelte

```diff
@@ -104,10 +104,10 @@
 </script>
 
 <svelte:element
-  this={href ? 'a' : 'button'}
+  this={href && !disabled ? 'a' : 'button'}
   {target}
-  {href}
-  disabled={href ? null : disabled}
+  href={disabled ? null : href}
```

Comment:
Might want to retain the check here if href is passed otherwise we may get `<button />`'s with an empty `href` I think?

Reply:
This should behave the same since `href` defaults to `null` if not passed, so it would be still be `href={null}` even if `disabled` is `false`.